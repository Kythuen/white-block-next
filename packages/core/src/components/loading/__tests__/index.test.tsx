import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import Loading from '../loading.vue'
// import { useLoading } from '../use'

describe('# Loading', () => {
  describe('## Base', () => {
    it('### default config', () => {
      const wrapper = mount({
        render() {
          return <Loading />
        }
      })
      // expect(wrapper.classes()).toEqual(['wb-loading'])
      // expect(wrapper.props()).toEqual({})
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  // describe('## Slots', () => {})

  // describe('## Events', () => {})

  // describe('## Props', () => {})

  // describe('## Composables', () => {})
})
