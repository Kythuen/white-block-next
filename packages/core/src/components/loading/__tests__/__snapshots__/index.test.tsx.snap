// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`# Loading > ## Base > ### default config 1`] = `
<div
  class="wb-loading"
>
   Loading 
  <!---->
</div>
`;
